Mail/Views/Thread/MessageBodyView.swift

```diff
@@ -85,14 +75,31 @@ struct MessageBodyView: View {
 
     private func loadBody(blockRemoteContent: Bool) {
         Task {
+            var messageBody = model.showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody
+
+            if presentableBody.body?.type == "text/plain" {
+                messageBody = createHtmlForPlainText(text: messageBody)
+            }
+
             let loadResult = await model.loadHTMLString(
-                value: model.showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody,
+                value: messageBody,
                 blockRemoteContent: blockRemoteContent
             )
 
             displayContentBlockedActionView = (loadResult == .remoteContentBlocked)
         }
     }
+
+    private func createHtmlForPlainText(text: String?) -> String {
+        do {
+            guard let text else { return "" }
+            let doc = try SwiftSoup.parse("")
```

Comment:
To create an empty document, you should be able to do something like `Document.createShell`